dask_jobqueue/core.py

```diff
@@ -498,6 +512,23 @@ def scale(self, n=None, jobs=0, memory=None, cores=None):
     def adapt(
         self, *args, minimum_jobs: int = None, maximum_jobs: int = None, **kwargs
     ):
+        """ Scale Dask cluster automatically based on scheduler activity.
+
+        Parameters
+        ----------
+
```

Comment:
I think we should also include the minimum/maximum arguments here. I suspect most users will interact with these regularly and I think chasing the arguments through to dask.distributed.Adaptive will end up being annoying.